python/spear/config/default_config.wheeled_vehicle.yaml

```diff
@@ -39,22 +32,28 @@ OPENBOT:
       YAW: 0.0                     # Sonar relative orientation in [°]
       ROLL: 0.0                    # Sonar relative orientation in [°]
 
-  OPENBOT_WHEEL:
+  VEHICLE_COMPONENT:
+    MASS: 1.12                   # Mass of the vehicle chassis in [kg]
+    DRAG_COEFFICIENT: 1.0        # DragCoefficient of the vehicle chassis
+    CHASSIS_WIDTH: 15.0          # Chassis width used for drag force computation in [cm]
+    CHASSIS_HEIGHT: 15.0         # Chassis height used for drag force computation in [cm]
+
+  VEHICLE_WHEEL:
     AFFECTED_BY_HANDBRAKE: True       # Whether handbrake should affect this wheel
-    DAMPING_RATE: 0.25                # Damping rate for this wheel (Kgm^2/s) 
     MAX_BRAKE_TORQUE: 1500.0          # Max brake torque for this wheel (Nm)
     MAX_HAND_BRAKE_TORQUE: 3000.0     # Max handbrake brake torque for this wheel (Nm). A handbrake should have a stronger brake torque than the brake.
+    
     LAT_STIFF_MAX_LOAD: 2.0           # Max normalized tire load at which the tire can deliver no more lateral stiffness no matter how much extra load
                                       # is applied to the tire.
     LAT_STIFF_VALUE: 17.0             # How much lateral stiffness to have given lateral slip
     LONG_STIFF_VALUE: 1000.0          # How much longitudinal stiffness to have given longitudinal slip
-    SHAPE_RADIUS: 3.3                 # Radius of the wheel in [cm]
-    SHAPE_WIDTH: 2.5                  # Width of the wheel in [cm]
-    STEER_ANGLE: 0.0                  # Steer angle in degrees for this wheel
+    
     SUSPENSION_MAX_RAISE: 0.0         # How far the wheel can go above the resting position in [m]
     SUSPENSION_MAX_DROP: 0.0          # How far the wheel can drop below the resting position in [m]
     SUSPENSION_FORCE_OFFSET: 0.0      # Vertical offset from where suspension forces are applied (along Z-axis) 
-    SUSPENSION_NATURAL_FREQUENCY: 7.0 # Oscillation frequency of suspension. Standard cars have values between 5 and 10
+    SUSPENSION_NATURAL_FREQUENCY: 7   # Oscillation frequency of suspension. Standard cars have values between 5 and 10
```

Comment:
7.0 is better because it's useful to communicate to the reader that it's a floating-point parameter.

Reply:
removed this parameter altogether as it is not used